main/FloatingActionButtonWrapper/index.tsx

```diff
@@ -0,0 +1,120 @@
+import {DoobooTheme, withTheme} from '../theme';
+import {Icon, IconName} from '../Icon';
+import React, {FC, useMemo} from 'react';
+
+import {Animated} from 'react-native';
+import {IconButton} from '../IconButton';
+import styled from '@emotion/native';
+
+const AbsolutePositionWrapperView = styled.View`
```

Comment:
I think `AbsolutePositionWrapperView` is too verbose. I'd rather use code below.
```js
<View style={{position: 'absolute'}}>
</View>
```

Considering further changes and abstraction, name it to `~Container` might be better,

Reply:
I think it is good thanks.  👍 
I thought I'd use more styles when I first added it.( But I didn't )

Reply:
but I think we should support default components such as <IconButton> with conditionals.
Because it makes it much easier to use components. It will improve developers' experience in use.

Reply:
@wndudqus  For sure. My code above support default components, but I remove conditional using default value which language provide.

```
// code from props destructing above
renderInactiveRootItem = defaultRootRenderer,
renderActiveRootItem = defaultRootRenderer,
renderSubItem = defaultSubRenderer,
```

Note that I set default render function that I defined above. User can provide render function, or omit it.